src/test/java/com/hubspot/jinjava/interpret/DeferredTest.java

```diff
@@ -119,6 +119,14 @@ public void itResolvesIfTagWherePossible() {
     assertThat(interpreter.getErrors()).isEmpty();
   }
 
+  @Test
+  public void itResolveEqualToInOrCondition() {
```

Comment:
Thanks for adding tests here too. Would you mind adding another for a deferred value on the RHS of an if?

Reply:
Sure @Joeoh. Before that, when you say RHS you mean right side of the equation, right? If so, can you give me an example about this test scenario to make clear I got your test idea?

Reply:
Yes the right hand side. Similar to itPreservesIfTag above we want to ensure that the if is preserved if a deferred variable is used in the Or condition. so for example: `{% if 'a' is equalto 'b' or 'a' is equalto deferred %}preserved{% endif %}

Reply:
done @Joeoh .

Reply:
Thanks! Can you use a var for the string instead

Reply:
pls, let me know if you have any other suggestion @Joeoh